state-chain/pallets/cf-flip/src/benchmarking.rs

```diff
@@ -28,9 +30,36 @@ mod benchmarks {
 	use super::*;
 
 	#[benchmark]
-	fn set_slashing_rate() {
+	fn on_initialize() {
+		let whitelisted_callers = T::AccountRoleRegistry::generate_whitelisted_callers_with_role(
```

Comment:
I think it would be better to benchmark the weight it takes to remove 1 storage, then calculate the weight using the actual number of accounts to remove, than to use an arbitrary number of 50